x-pack/plugins/infra/server/lib/alerting/inventory_metric_threshold/inventory_metric_threshold_executor.ts

```diff
@@ -85,31 +97,36 @@ export const createInventoryMetricThresholdExecutor = (libs: InfraBackendLibs) =
         logger.error(e.message);
         const actionGroupId = FIRED_ACTIONS.id; // Change this to an Error action group when able
         const reason = buildInvalidQueryAlertReason(params.filterQueryText);
-        const alert = alertFactory('*', reason);
-        const indexedStartedDate = getAlertStartedDate('*') ?? startedAt.toISOString();
+        const alert = alertFactory(UNGROUPED_FACTORY_KEY, reason);
+        const indexedStartedDate =
+          getAlertStartedDate(UNGROUPED_FACTORY_KEY) ?? startedAt.toISOString();
+        const alertUuid = getAlertUuid(UNGROUPED_FACTORY_KEY);
         const viewInAppUrl = getViewInAppUrlInventory(
           criteria,
           nodeType,
           indexedStartedDate,
           libs.basePath
         );
+
         alert.scheduleActions(actionGroupId, {
-          group: '*',
+          alertDetailsUrl: getViewInAppUrl(libs.basePath, `${LINK_TO_ALERT_DETAIL}/${alertUuid}`),
```

Comment:
Nit: As the usage of `getViewInAppUrl` is more generic now, I suggest renaming it.